common/test/acceptance/pages/lms/video/video.py

```diff
@@ -57,7 +57,8 @@
     'language': '.lang .menu',
     'speed': '.speed .menu',
     'download_transcript': '.video-tracks .a11y-menu-list',
-    'transcript-format': '.video-tracks .a11y-menu-button'
+    'transcript-format': '.video-tracks .a11y-menu-button',
+    'transcript-skip': '.sr-is-focusable.transcript-start'
```

Comment:
I'd add a comma at the end of the last line in this dict. (See [why-does-python-allow-commas-at-the-end-of-lists-and-tuples](https://docs.python.org/2/faq/design.html#why-does-python-allow-commas-at-the-end-of-lists-and-tuples).)